### ISSUE TITLE
Network.vertical isn't implemented

### ISSUE THREAD

**Issue by cliffckerr** [starsimhub/starsim#491]
Maybe I'm missing something, but I don't see anywhere where setting the `vertical` arg actually changes the transmission logic

**Comment by cliffckerr** [comment]
It is partly implemented, but seemingly still relies on beta for the child → mother route being manually set to 0.

**Comment by kaa2102** [comment]
@cliffckerr sim.demographics.Pregnancy() and sim.networks['mfnet'] have arrays for p1 (mothers), p2 (unborn babies), beta, and the edge p1 - p2. However, these are not used and factored into Births. I'm updating Births to align with the behavior of Deaths, Pregnancies. I'm creating a properties and logic in sim.demographics to represent mothers, babies and ti of child birth.

**Comment by cliffckerr** [comment]
Closed by #509
